generators/app/index.js

```diff
@@ -136,6 +136,22 @@ module.exports = Generator.extend({
       );
     },
 
+    generateProfile: function() {
+      if (options.projectProfile != undefined && options.projectProfile != 'none') {
```

Comment:
After the prompt changes I recommended, this can be a simpler `if (options['generateProfile'])` statement, given `generateProfile` is the boolean prompt name.